javascript/stimulus_reflex.js

```diff
@@ -90,8 +88,8 @@ const createSubscription = controller => {
                 )
               )
             )
-            if (urls.length !== 1 || urls[0] !== location.href) return
 
+            if (urls.length !== 1 || urls[0] !== location.href) return
```

Comment:
What changed on this line? I'm trying to understand why the diff shows this as different.

Reply:
Oh, I think it's because I deleted the empty line between this one and the next. 🤷‍♂️